lang/src/org/partiql/lang/eval/ExprValueExtensions.kt

```diff
@@ -100,9 +100,9 @@ fun ExprValue.stringValue(): String =
 fun ExprValue.bytesValue(): ByteArray =
     scalar.bytesValue() ?: errNoContext("Expected LOB: $ionValue", internal = false)
 
-internal fun ExprValue.datePartValue(): DatePart =
+internal fun ExprValue.datePartValue(): DateTimePart =
```

Comment:
May also want to change the name of this extension function to `dateTimePartValue()`?